src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html

```diff
@@ -89,9 +89,9 @@
 
             <span class="DevHub-MyAddons-item-modified">
               {% if addon.last_updated %}
-                {{ _('Last Updated:') }}
+                <span>{{ _('Last Updated:') }}</span>
                 {# L10n: {0} is a date. dennis-ignore: E201,E202,W202 #}
```

Comment:
This comment doesn't appear to make sense, but I wonder would a placeholder be a better solution for this?